assets/sass/okta-theme.scss

```diff
@@ -57,7 +57,7 @@
 
 
 // scss-lint:disable IdSelector
-#okta-sign-in {
+#okta-sign-in.auth-container {
```

Comment:
The problem is that we changed the specificity of the other classes to be at the ID level. Originally the ID level was enough to override other styles, but now we have to add the class to increase the specificity.